core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -145,6 +152,23 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
+    private void printMetrics(StringBuilder output, String label, Metrics metrics)
+    {
+        if (!verbose || metrics.getMetrics().isEmpty()) {
+            return;
+        }
+        output.append(label).append("\n");
+        Map<String, Metric> sortedMap = new TreeMap(metrics.getMetrics());
+        sortedMap.forEach((name, metric) -> {
+            if (metric instanceof Count) {
+                output.append(format("  '%s' = %s\n", name, ((Count) metric).getTotal()));
+            }
+            if (metric instanceof Distribution) {
```

Comment:
it seems that `TDigestHistogram` is missing `toString`.

Is there a nice way to print percentiles (e.g: for 0, 10, 20, ..., 90, 100)

Reply:
Added in a separate commit.